src/nexmo/verify.py

```diff
@@ -0,0 +1,89 @@
+import nexmo
+import warnings
+
+class Verify:
+    def __init__(
+        self,
+        client=None,
+        key=None,
+        secret=None
+    ):
+        try:
+            self._client = client
+            if self._client is None:
+                self._client = nexmo.Client(
+                    key=key,
+                    secret=secret
+                )
+        except Exception as e:
+            print('Error: {error_message}'.format(error_message=str(e)))
+    
+    def request(self, params=None, **kwargs):
+        return self._client.post(self._client.api_host(), "/verify/json", params or kwargs)
+    
+    def send_verification_request(self, params=None, **kwargs):
+        warnings.warn(
+            "nexmo.Client#send_verification_request is deprecated (use #start_verification instead)",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+
+        return self._client.post(self._client.api_host(), "/verify/json", params or kwargs)
+    
+    def check(self, request_id, params=None, **kwargs):
+        return self._client.post(
+            self._client.api_host(),
+            "/verify/check/json",
+            dict(params or kwargs, request_id=request_id),
+        )
+    
+    def check_verification_request(self, params=None, **kwargs):
```

Comment:
looks like we dropped a `check_verification` method from the original module. It looks like that was meant to be the 'real' check_verification request. I say that as this one has been marked as deprecated.